### PR TITLE
[Documentation update] Update how-to-run-example-android.md @open sesame 05/16 20:21

### PR DESCRIPTION
**Pull request by udit01**
Updated documentation to include some additional things needed and added some troubleshooting cases.

Signed-off-by: Udit <udit.jain@samsung.com>

### CODE REVIEW

File: docs/how-to-run-example-android.md [hunk 1]
```diff
@@ -85,3 +87,15 @@ Now you can find the execution binary of nntrainer_logistic.
 You can copy execution binary and nntrainer.so libraries in a proper place of your android device using ADB.
 
 Then you can run it using ADB shell.
+
+
+### Troubleshooting
+
+1. Check version of your ndk before executing the above commands.
+2. `ANDROID_NDK` not defined while bulding the particular application such as Logistic regression.
```
Comment (lhs8928) [inline docs/how-to-run-example-android.md:95]: typo: bulding

**Review by lhs8928 — APPROVED** [review]
LGTM

**Comment by jijoongmoon** [comment]
Could you elaborate on what the differences are between #1907 and #1906? If those are the same, then could you merge them?

**Comment by udit01** [comment]
Yes, I tried to follow the "Signed off by" and other formatting in #1907, I don't think I have permissions to delete #1906. 
It should be just one PR #1907 .

**Comment by jijoongmoon** [comment]
It is better if you using git commit msg.txt.
here is the example
```
Commit title (Until 50 colums per line)

Describe a commit content (Until 80 colums per line) in detail ASAP.

**Changes proposed in this PR:**
- Added TOC generator for README.md

Resolves:

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

```
For the signed off, 
you could use ``` git commit --signoff ```

**Comment by udit01** [comment]
Thank you for the explanation!
 I'll use the instructions when I make a PR next time.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**